### ISSUE TITLE
[BUG] Buttons lack sufficient contrast

### ISSUE THREAD

**Issue by sublimemarch** [ContributorCovenant/beacon#64]
All contributions, including pull requests, issues, and comments, are governed by our [code of conduct](https://github.com/ContributorCovenant/beacon/blob/release/CODE_OF_CONDUCT.md).

## Description of the bug
Orange buttons throughout the site don't have enough contrast, per [WCAG 2.1 AA guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html). This was determined using [Axe](https://www.deque.com/axe/) and can be verified using any color contrast checking tool, such as the [WebAIM color contrast checker](https://webaim.org/resources/contrastchecker/).

The white text against orange has a contrast ratio of 3.4 curently. The expected contrast ratio for text of this size is 4.5.

## Expected behavior
The colors of these elements should be updated to have more contrast. Alternatively, a bolder font or larger font size could be used, but I'd recommend updating the colors themselves. This is a necessary accessibility improvement for people with disabilities (such as color blindness and low-vision).

## Screenshots
This applies everywhere that orange buttons are used with white text, for example:
![image](https://user-images.githubusercontent.com/9502152/52145802-af693b00-2627-11e9-9833-903129cd2389.png)
